app/controllers/dashboards_controller.rb

```diff
@@ -1,11 +1,13 @@
-# typed: true
+# typed: false
 # frozen_string_literal: true
 
 # Displays the list of collections to the user
 class DashboardsController < ApplicationController
   before_action :authenticate_user!
+  verify_authorized
 
   def show
     @collections = authorized_scope(Collection.all)
+    authorize! :dashboard
```

Comment:
I don't think it's a big deal, but I wonder if we need line 11 below line 10 here. (Not familiar with `authorized_scope`.)